user/nemo/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "nemo"
+pkgver = "6.2.8"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dgtk_doc=true", "-Dtracker=true"]
+hostmakedepends = [
+    "gobject-introspection",
+    "gtk-doc-tools",
+    "intltool",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "cinnamon-desktop-devel",
+    "exempi-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "json-glib-devel",
+    "libexif-devel",
+    "libgsf-devel",
+    "libx11-devel",
+    "libxml2-devel",
+    "pango-devel",
+    "tracker-devel",
+    "xapp-devel",
+]
+pkgdesc = "File browser for Cinnamon"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later AND GFDL-1.1-or-later AND LGPL-2.0-or-later"
+url = "https://projects.linuxmint.com/cinnamon"
+source = f"https://github.com/linuxmint/nemo/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "68d829614beeb2596aa706db5ff070fcfc0e96ab7e3d2561772b52bed6c8ca38"
+# Tests require a graphical environment
```

Comment:
this exists, see make_check_wrapper wlheadless-run use in the repo

since this is gtk3, you will probably need `"wlheadless-run", "-c", "cage", "--"` as the invocation

Reply:
Tests require its own GSettings schemas to be installed, I will not be enabling the "check" phase.

Reply:
there's env vars for that, but the bigger issue is that the nemo tests are broken entirely for other reasons (known upstream). i just mean to show you that "need graphical env" is not actually the blocker :)